lib/go-tc/deliveryservice_ssl_keys.go

```diff
@@ -73,6 +73,7 @@ type SSLKeyRequestFields struct {
 	HostName     *string `json:"hostname,omitempty"`
 	Country      *string `json:"country,omitempty"`
 	State        *string `json:"state,omitempty"`
+	Version      *int    `json:"version,omitempty"`
```

Comment:
I'm not sure we can add this without making a new-version structure, because this is used in the APIv3 client method `GenerateSSLKeysForDS` which has been published. That's what @rob05c would say, I think. @rawlinp will disagree, saying that it won't break existing clients because they couldn't have been using it anyway - which is true.

Personally I don't care either way, but would find it easier to just make an `SSLKeyRequestV40` and `SSLKeyRequestV4` alias as it's a small enough change and won't hurt anything even if it's unnecessary. But if rob doesn't care I don't think anyone else will, so in that case don't worry about it.

Reply:
I believe we are free to add new fields to existing structs -- that is a non-breaking change.

Reply:
> I believe we are free to add new fields to existing structs -- that is a non-breaking change.

Not only that, but it looks like the "new" field was used in older API versions already, the only difference is that the Go client used to hard-code it. So this is barely a **non**-breaking change.

Reply:
+1 - as long as it has a sane default, ideally as close to the prior behavior as possible.

It technically can break code, e.g. `foo := Foo{a,b,c}` will break. But it's reasonable to add fields in minor versions, generally speaking, and any of the alternatives in Go aren't reasonable.

So, as the person who's usually one of the biggest votes against breaking clients, IMO we should consider adding fields as non-breaking, and discourage clients from using that syntax (and the few others which break, which all have alternatives that don't).